src/vm-vcpu/src/vcpu/mod.rs

```diff
@@ -684,7 +684,17 @@ impl KvmVcpu {
                                 }
                             } else if addr == 0x060 || addr == 0x061 || addr == 0x064 {
                                 // Write at the i8042 port.
+                                //i8042 is registered at port 0x64.
                                 // See https://wiki.osdev.org/%228042%22_PS/2_Controller#PS.2F2_Controller_IO_Ports
+                                if self
```

Comment:
This is not exactly correct. Even though for the writes that result in cpu exit, it is expected that `addr` will be 0x64, we shouldn't make any assumptions at this point about the exact value. A driver can also write to 0x60, see example [here](https://github.com/firecracker-microvm/firecracker/blob/main/src/devices/src/legacy/i8042.rs#L290), but we don't offer this support in vm-superio.
It is enough to just use `addr` instead of the hardcoded value below, and the i8042 logic will know how to handle different addresses (at this point it does nothing for addresses != 0x64).